mautic/mautic
# Disable search indexing with noindex on form page

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| Related user documentation PR URL | [mautic/documentation#282](https://github.com/mautic/documentation/pull/282)
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Add button "Disable search indexing" to block search indexing with 'noindex' on form page https://support.google.com/webmasters/answer/93710?hl=en

![screenshot-2018-6-4 edit form - contact mautic](https://user-images.githubusercontent.com/27768270/40909876-ff79264c-67ea-11e8-9f43-3526840bf472.png)


[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply PR and test migrations `php app/console doctrine:migrations:migrate`
2. Create form page and see in right column new option to Disable search indexing
3. Enable it and test it in Noindex checker https://seositecheckup.com/tools/noindex-checker
